docs/exp-algo/pagerankv.md

```
@@ -0,0 +1,110 @@
+It supports **weighted transitions**: rank can be distributed by:
+- a single edge property (default: `weight`)
+- first-found among multiple property names
+- a weighted-sum map of edge properties
+
+## Syntax
+```cypher
+flex.exp.pagerankv({nodes: nodes, direction: 'both', damping: 0.85, maxIterations: 50, tolerance: 1e-8, personalization: {'42': 1}})
+```
+
+## Parameters
+| Parameter | Type | Required | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)